Build error on openSUSE Tumbleweed: expected (, got newline and text

Hi,

On my i586 openSUSE Tumbleweed openSUSE Build Service (OBS) build target I get the cmake error:

```bash
[  135s] -- Detecting CXX compiler ABI info
[  136s] -- Detecting CXX compiler ABI info - done
[  136s] -- Detecting CXX compile features
[  139s] -- Detecting CXX compile features - done
[  139s] -- Performing Test HAVE_GCC_ERROR_RETURN_TYPE
[  139s] -- Performing Test HAVE_GCC_ERROR_RETURN_TYPE - Success
[  139s] -- Could NOT find Git (missing: GIT_EXECUTABLE) 
[  139s] -- Could not use git to determine source version, using version 1.90.0
[  139s] -- Found Eigen3: /usr/include/eigen3 (Required is at least version "2.91.0") 
[  139s] -- Found Doxygen: /usr/bin/doxygen (found version "1.8.14") found components:  doxygen missing components:  dot
[  139s] CMake Error at cmake/AvogadroCPack.cmake:37:
[  139s]   Parse error.  Expected "(", got newline with text "
[  139s] 
[  139s]   ".
[  139s] Call Stack (most recent call first):
[  139s]   CMakeLists.txt:73 (include)
[  139s] 
[  139s] 
[  139s] -- Configuring incomplete, errors occurred!
[  139s] See also "/home/abuild/rpmbuild/BUILD/avogadroapp-b1838d884908acfbcc2a1f61259e8bea78be811e/build/CMakeFiles/CMakeOutput.log".
[  139s] error: Bad exit status from /var/tmp/rpm-tmp.MluVGZ (%build)
[  139s] 
```

(ignore the `[139s]`, it pertains to the build time in the OBS). The cmake command used is:

```bash
cmake -DCMAKE_BUILD_TYPE:STRING=Release \ 
 -Wno-dev \ 
 -DCMAKE_VERBOSE_MAKEFILE:BOOL=TRUE \ 
 -DENABLE_RPATH:BOOL=ON \ 
 -DENABLE_TESTING:BOOL=OFF \ 
 -DAvogadroLibs_DIR:PATH=/usr/lib32 \ 
 -DBUILD_DOCUMENTATION:BOOL=ON .. 
```

This is an error I only have gotten with the latest git commit #407, b1838d8. Earlier commits built fine. Full log is [here](https://gist.github.com/b6e1fa656a5f1b777e00d73f2c16d1fe).

Thanks for your time and patience,
Brenton

--- a/cmake/AvogadroCPack.cmake
+++ b/cmake/AvogadroCPack.cmake
@@ -33,7 +33,7 @@ if(APPLE)
   set(INSTALL_BUNDLE_FILES ON)
 else()
   option(INSTALL_BUNDLE_FILES "Add install rules to bundle files" OFF)
-endif
+endif()
 if(INSTALL_BUNDLE_FILES)
   # First the AvogadroLibs files that are not detected.
   find_package(AvogadroLibs REQUIRED NO_MODULE)

